### DISCUSSION TITLE
OpenWrt上让你的四级设备获取到能正常使用IPv6公网地址 - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by giscus** [Muska-Ami/muska-ami-blog#71] [General]
# OpenWrt上让你的四级设备获取到能正常使用IPv6公网地址 - 夏沫花火zzz🌙 (Muska_Ami)のLife

环境说明环境为 光猫拨号，下面挂一个OpenWrt主路由，下级也挂OpenWrt主路由，下级一个AP需要使得第二个OpenWrt设备能正常获取IPv6公网地址并使得下挂设备同时正常获取IPv6公网地址 第一OpenWrt主路由配置wan6配置DHCP服务器 选项卡中点击配置DHCP服务器  常规设置 勾选 忽略此接口 IPv6设置 勾选 指定的主接口 RA服务 修改为 中继模式 DHCPv6服务

https://blog.1l1.icu/2024/04/20/openwrt-shang-rang-ni-de-si-ji-she-bei-huo-qu-dao-neng-zheng-chang-shi-yong-ipv6-gong-wang-di-zhi/

**Comment by ltzXiaoYanMo** [comment]
喵

**Reply by Muska-Ami** [reply]
喵
